src/mergecil.ml

```diff
@@ -65,11 +65,11 @@ let usePathCompression = false
 (* Try to merge definitions of inline functions. They can appear in multiple
  * files and we would like them all to be the same. This can slow down the
  * merger an order of magnitude !!! *)
-let mergeInlines = true
+let merge_inlines = ref false
 
-let mergeInlinesRepeat = mergeInlines && true
+let mergeInlinesRepeat = !merge_inlines && true
 
-let mergeInlinesWithAlphaConvert = mergeInlines && true
+let mergeInlinesWithAlphaConvert = !merge_inlines && true
```

Comment:
These don't actually account for `merge_inlines` being changed later.

Reply:
I changed them to be functions taking unit.